src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,33 +2,39 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    private static final int EMPTY_FIFO = -1;
+    private static final int FULL_FIFO = 11;
+    private static final int CAPACITY = 12;
+    private int[] numbers = new int[CAPACITY];
 
-    public int total = -1;
+    private int total = EMPTY_FIFO;
 
     public void countIn(int in) {
         if (!isFull())
             numbers[++total] = in;
     }
 
-    public boolean callCheck() {
-        return total == -1;
+    boolean callCheck() {
+        return total == EMPTY_FIFO;
     }
 
-    public boolean isFull() {
-        return total == 11;
+    boolean isFull() {
+        return total == FULL_FIFO;
     }
 
-    protected int peekaboo() {
+    int peekaboo() {
         if (callCheck())
-            return -1;
+            return EMPTY_FIFO;
```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)